src/views/3-Modeling/Modeling.vue

```diff
@@ -369,19 +806,27 @@
           // set up flubber svg
           setFlubber() {
             const self = this;
-            // display only the ANN path elements
-            self.d3.select("#transform-svg-test")
-                .selectAll("#RNN")
-                  .remove();
-            self.d3.select("#transform-svg-test")
-                .selectAll("#RGCN")
-                  .remove();
-            self.d3.select("#transform-svg-test")
-                .selectAll("#RGCN_2")
-                  .remove();
-            self.d3.select("#transform-svg-test")
-                .selectAll("#RGCN_ptrn")
-                  .remove();
+
+            // determine initial model id and initial annotation id
+            // NOTE currently assumes that we are beginning the visuals at step 0
+            let initial_model_id = self.flubber_id_order[self.step]
+            let initial_annotation_id = initial_model_id + "_annotations"
+
+            // Hide all flubber elements (visuals and annotations)
+            self.d3.selectAll(".flubber")
+              .attr("opacity", 0)
+              // .style("visibility", "hidden");
+
+            // display visual associated with initial model id
+            self.d3.selectAll("#" + initial_model_id)
+              .attr("opacity", 1)
+              // .style("visibility", "visible");
+
+            // display visual associated with initial model id
+            self.d3.selectAll("#" + initial_annotation_id)
+              .attr("opacity", 1)
+              // .style("visibility", "visible");
+
```

Comment:
this is great. would be useful to set this up as a function that could be applied to different sections, like for the beeswarm to be able to check if the x-axis variable is the same or different than the previous step, and if not, to run the tick function. that would help me fix where it jiggles as each step passes, even though the variable stays the same. I'll look back to this when I'm working on that

Reply:
another thought about the self.step - if we administer this as a computed property then it caches, so it will maintain the last value if the page refreshes. This would be great because then all elements are rendered based on the current step and it wouldn't redraw the whole chart and then get to the right step (as currently is). Similarly, I think if the initial chart/core elements of it are being drawn in a `created()` call, it will only be drawn at the first initialization. pipe dreams

Reply:
I'm not asking you to do these things, just thinking out loud here

Reply:
Ohh, that's cool that we could used the cached value of `self.step` to fix that issue with rendering upon page refresh. Good thinking. Let's definitely circle back to that.